server/src/main/java/com/netflix/conductor/server/ConductorServer.java

```diff
@@ -124,15 +126,15 @@ private void init(String dynoClusterName, List<Host> dynoHosts) {
 		case dynomite:
 			ConnectionPoolConfigurationImpl connectionPoolConfiguration = new ConnectionPoolConfigurationImpl(dynoClusterName)
 					.withTokenSupplier(getTokenMapSupplier(dynoHosts))
-					.setLocalRack(conductorConfig.getAvailabilityZone())
-					.setLocalDataCenter(conductorConfig.getRegion())
+					.setLocalRack(systemPropertiesConfiguration.getAvailabilityZone())
+					.setLocalDataCenter(systemPropertiesConfiguration.getRegion())
 					.setSocketTimeout(0)
 					.setConnectTimeout(0)
-					.setMaxConnsPerHost(conductorConfig.getIntProperty("workflow.dynomite.connection.maxConnsPerHost", 10));
+					.setMaxConnsPerHost(systemPropertiesConfiguration.getIntProperty("workflow.dynomite.connection.maxConnsPerHost", 10));
```

Comment:
**Question**: Couldn't the `getMaxConnectionsPerHost()` be used here?

Reply:
Not really, because this is the parent type `Configuration`, not `DynomiteConfiguration`.  In this location forcing `DynomiteConfiguration` seems like an overly strong dependency since you might not be using Dynomite at all.

The good news is that I think this is mostly addressed in my followup PR gorzell/conductor/pull/5.